src/routes/components/components.json

```
@@ -2676,5 +2676,14 @@
 		"category": "User Interaction",
 		"tags": ["animations", "components and libraries", "interactions"],
 		"stars": 0
-	}
+	},
+  	{
+    		"title": "svelte-switch-case",
```

There are spaces instead of a tabulation
```suggestion
		"title": "svelte-switch-case",
```